sites/partners/src/lib/users/signInHelpers.ts

```diff
@@ -3,19 +3,55 @@ import { MfaType } from "@bloom-housing/shared-helpers/src/types/backend-swagger
 export enum EnumRenderStep {
   emailAndPassword = "email and password",
   mfaType = "mfa type",
-  phoneNumber = "phone number if missing",
+  phoneNumber = "phone number is missing",
   enterCode = "enter mfa code",
 }
 
 export const onSubmitEmailAndPassword =
```

Comment:
We should probably move this out of this file w all the prop drilling, but didn't want to overhaul that here